spec/ruby/optional/capi/ext/io_spec.c

```diff
@@ -28,9 +28,13 @@ static int set_non_blocking(int fd) {
 }
 
 static int io_spec_get_fd(VALUE io) {
+#ifdef RUBY_VERSION_IS_3_3
```

Comment:
Should be 3_1 since it exists since 3.1, I changed it in 957b6a491fe6a9a2df467f83a37e7c34861f995c

Reply:
Yes, but this will break truffle ruby right? That's why I changed it to 3.3 to give you time to implement it.. unless you want to backport it? IIUC.